modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/RestGeoIpDownloaderStatsAction.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.ingest.geoip;
+
+import org.elasticsearch.action.ActionListener;
+import org.elasticsearch.action.admin.cluster.node.tasks.list.ListTasksAction;
+import org.elasticsearch.action.admin.cluster.node.tasks.list.ListTasksRequest;
+import org.elasticsearch.client.node.NodeClient;
+import org.elasticsearch.common.xcontent.ToXContentObject;
+import org.elasticsearch.rest.BaseRestHandler;
+import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.rest.action.RestToXContentListener;
+import org.elasticsearch.tasks.TaskInfo;
+
+import java.util.List;
+
+import static org.elasticsearch.rest.RestRequest.Method.GET;
+
+public class RestGeoIpDownloaderStatsAction extends BaseRestHandler {
+
+    @Override
+    public String getName() {
+        return "geoip_downloader_stats";
+    }
+
+    @Override
+    public List<Route> routes() {
+        return List.of(new Route(GET, "_geoip/stats"));
+    }
+
+    @Override
+    protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient client) {
+        ListTasksRequest listTasksRequest = new ListTasksRequest()
```

Comment:
I think should be done in a transport action (can be as simple as `HandledTransportAction`), the rest layer should just be a layer that translates a rest request to an action request and delegates to transport action.

This can make applying more fine grained security privileges possible (otherwise we always need to grant privileges on the list task action, which exposes a lot of other information) and this 
approach we do for all apis/actions, even if these actions are very simple.

Reply:
I've added relevant actions